dotcom-rendering/src/web/components/SignInGate/gateDesigns/shared.tsx

```diff
@@ -116,6 +116,7 @@ export const privacyLink = css`
 	text-underline-position: under;
 	border: 0;
 	background: transparent;
+	font-family: inherit;
```

Comment:
Seeing this used elsewhere in the code a few places in links and link buttons such as this:
```
/* stylelint-disable-next-line property-disallowed-list */
font-family: inherit;
```